src/main/java/uk/gov/hmcts/ccd/endpoint/exceptions/RestExceptionHandler.java

```diff
@@ -50,6 +60,29 @@ public RestExceptionHandler(AppInsights appInsights) {
                 .body(error);
     }
 
+    @ExceptionHandler(CaseValidationException.class)
+    @ResponseBody
+    public ResponseEntity<HttpError> handleCaseValidationException(HttpServletRequest request, CaseValidationException exception) {
+
+        // read field IDs from CaseValidationException.details (i.e. avoid using full error details as this may contain user data)
+        List<String> fieldIds = new ArrayList<>();
+        if (exception.getDetails() != null) {
+            String detailsJson = this.objectMapperService.convertObjectToString(exception.getDetails());
+            fieldIds = JsonPath.read(detailsJson, "$..field_errors[*].id");
+        }
```

Comment:
wonder whether we can enhance CaseValidationException with a property that stores fields with validation error so that here rather than manipulating JSON we could just call: `exception.getFields()` and we get back a list of field ids